client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/hooks/use-recipe.ts

```diff
@@ -73,12 +74,18 @@ const useRecipe = (
 
 	const { stylesheet = '' } = selectedDesign?.recipe || {};
 
+	const base_variation_stylesheet = isEnabled( 'design-picker/use-assembler-styles' )
+		? 'pub/assembler'
+		: '';
+
 	const colorVariations = useColorPaletteVariations( siteId, stylesheet, {
 		enabled: !! preselectedColorVariationTitle,
+		base_variation_stylesheet,
 	} );
 
 	const fontVariations = useFontPairingVariations( siteId, stylesheet, {
 		enabled: !! preselectedFontVariationTitle,
+		base_variation_stylesheet,
```

Comment:
I found that the Design Picker is still showing Creatio 2 fonts, unless we also pass this param on:

https://github.com/Automattic/wp-calypso/blob/78d8b6d04284bba9d2f76576bc65da7177db367b/packages/global-styles/src/components/font-pairing-variations/index.tsx#L94

(and maybe similarly for colors)

Could we double check?

Reply:
Or maybe we can put this logic inside the `useFontPairingVariations()` instead.

Reply:
Oopss, you're right! Fixed it by https://github.com/Automattic/wp-calypso/pull/89625/commits/f1258887c65a185711afc4d07204bc803b9774fa